specs/lib/loadImageUrisSpec.js

```diff
@@ -0,0 +1,88 @@
+var filePath = './specs/data/boxTexturedUnoptimized/';
+
+describe('loadImageUris', function() {
+    var imageData;
+
+    beforeAll(function(done) {
+        fs.readFile(imagePath, function (err, data) {
+            if (err) {
+                throw err;
+            }
+            imageData = data;
+            done();
+        });
+    });
+    
+    it('loads an external image', function(done) {
+        var gltf = {
+            "images": {
+                "Image0001": {
+                    "uri": "Cesium_Logo_Flat.png"
+                }
+            }
+        };
+
+        gltf = loadGltfUris(gltf, filePath, function() {
+            expect(gltf.images.Image0001.extras.source).toBeDefined();
+            expect(gltf.images.Image0001.extras.source.toString()).toEqual(imageData.toString());

```

Comment:
Instead of comparing these as strings, can we compare the raw bytes?  It is more direct.  See if there is a `memcmp` type function for buffers.